packages/tldraw/src/lib/shapes/note/NoteShapeUtil.tsx

```diff
@@ -296,36 +321,76 @@ export class NoteShapeUtil extends ShapeUtil<TLNoteShape> {
 }
 
 function getGrowY(editor: Editor, shape: TLNoteShape, prevGrowY = 0) {
+	const unadjustedFontSize = LABEL_FONT_SIZES[shape.props.size]
+	const { labelHeight, fontSizeAdjustment } = getLabelSize(editor, shape)
```

Comment:
The geometry is cached, so we could ask the editor (ie `editor.getShapeGeometry(shape).children[`]) rather than recalculate this?